packages/core/README.md

```diff
@@ -20,7 +20,7 @@
 
 Page editors are a great way to provide an excellent user experience. However, to build one is often a pretty dreadful task.
 
-There're existing libraries that come with a fully working page editor out of the box with a user interface and editable components. However, if you wish to make customisations such as modifying the user interface and its behavior, it will most definitely involve modifying the library itself.
+There are existing libraries that come with a fully working page editor out of the box with a user interface and editable components. However, if you wish to make customisations such as modifying the user interface and its behavior, it will most definitely involve modifying the library itself.
```

Comment:
I think we can keep the original for this one

Reply:
Reverted to "There're"  in https://github.com/prevwong/craft.js/pull/129/commits/b93089054b98553e523d3a0b34f91d94e8dc19d9

Thx!!!